relationships.md

```diff
@@ -0,0 +1,160 @@
+        },
+        "rightType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+        },
+        "self": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+        }
+      },
+      "_embedded": {
+        "leftType": {
+          "id": 1,
+          "label": "Publication",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1/relationshiptypes"
+            }
+          }
+        },
+        "rightType": {
+          "id": 2,
+          "label": "Person",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The [relationship type](relationshiptypes.md) is embedded
+The 2 items are included as HAL links but are not embedded
+
+## Creating a relationship
+
+**POST /api/core/relationships?relationshipType=<:relationshipType>**
+
+A new relationship between 2 items can be created by specifying both items in a uri-list body and specifying the relationshipType as a parameter
+
+A sample CURL command would be:
+```
+curl -i -X POST 'https://dspace7-entities.atmire.com/rest/api/core/relationships?relationshipType=1' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
+```
+
+The uri-list should always contain exactly 2 items. The first item will be used as the left Item. The second item will be used as the right Item.
+The relationshipType parameter is mandatory as well
+
+## Updating a relationship
+
+**PUT /api/core/relationships/<:id>?relationshipType=<:relationshipType>**
+
+Update the items in the relationship
+
+A sample CURL command would be:
+```
+curl -i -X PUT 'https://dspace7-entities.atmire.com/rest/api/core/relationships/891' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
+```
+
+The uri-list should always contain exactly 2 items. The first item will be used as the left Item. The second item will be used as the right Item.
+The relationshipType is not modifiable
+
+## Relationships per item
+**/api/core/items/<:uuid>/relationships**
```

Comment:
This is an Item endpoint. It should be described in `item.md` and not in `relationships.md`

Reply:
I've merged the changes now